src/main/java/com/example/demo/model/User.java

```diff
@@ -0,0 +1,40 @@
+package com.example.demo.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Data
+@Getter
+@Setter
+@Entity
+@SQLDelete(sql = "UPDATE users SET is_deleted = TRUE WHERE id = ?")
+@Where(clause = "is_deleted = FALSE")
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false, unique = true,name = "email")
+    private String email;
+    @Column(nullable = false,name = "first_name")
+    private String firstName;
+    @Column(nullable = false,name = "last_name")
```

Comment:
always format your files with IDEA's formatter (Ctrl Alt L to trigger it)
there should be a space after each comma